draft-ietf-detnet-dp-sol-mpls-02.xml

```diff
@@ -484,51 +576,107 @@ System  |    +--------+       +--------+       +--------+   | System
      <--IP-->| <-------- IP Over DetNet MPLS ---------> |<--IP-->
              |                                          |
              |<------ End to End DetNet Service ------->|
+
+    X   = Optional service protection (none, PRF, PREOF, PEF/POF)
+    DFx = DetNet member flow x over a TE LSP
 ]]></artwork>
 </figure>
- <t>
-  <xref target="fig_ip_pw_detnet"/> illustrates how end to end DetNet
-  service is provided where the end systems are able to send and receive
-  IP DetNet flows, e.g., per <xref target="I-D.ietf-detnet-dp-sol-ip"/>, and
-  the MPLS  nodes optionally provide service protection. In this
-  case R1 and R3 are T-PEs and R2 is an S-PE and the DetNet service is
-  end-to-end.
- </t>
+  </section>
+  <section title="IEEE 802.1 TSN Over MPLS DetNet Data Plane Scenario"

```

Comment:
This is a more general case. It can be any L2 not only TSN. For example if the IP End-systems are in the same subnet and connected via the MPLS domain. Worth to add a note?